src/panel/details.tsx

```diff
@@ -72,9 +72,8 @@ interface DetailsProps { result: Result, resultsFixed: string[], height: IObserv
                             <span>Rule Id</span>			{helpUri ? <a href={helpUri} target="_blank" rel="noopener noreferrer">{result.ruleId}</a> : <span>{result.ruleId}</span>}
                             <span>Rule Name</span>			<span>{result._rule?.name ?? '—'}</span>
                             <span>Rule Description</span>	<span>{renderRuleDesc(result)}</span>
+                            <span><strong>Hint</strong></span>				<span>{result._rule?.help?.text ?? '—'}</span>
```

Comment:
I don't think there is a need for strong emphasis. Let's also call this "Remediation" so that we stick with consistent terminology between Cloud IDE <> VSC Extension.